server/services/communityApiService.ts

```diff
@@ -0,0 +1,33 @@
+import type HmppsAuthClient from '../data/hmppsAuthClient'
+import RestClient from '../data/restClient'
+import config from '../config'
+import logger from '../../log'
+
+interface DeliusUser {
```

Comment:
is this an appropriate place for these ?

Reply:
I think we briefly talked about this the other day when we were wondering why that `types.ts` file existed in the repo George was showing us. It probably comes down to whether we think that every file that uses `DeliusUser` is also going to use `CommunityApiService`. Like, if `DeliusUser` is going to end up being passed between files all around the app, then do we want all of those files to have to import `CommunityApiService` even if they're not actually making API calls? (The answer may be "yes, that's fine", in which case this seems like a fine place to have these types)